.github/workflows/main.yml

```diff
@@ -253,7 +253,11 @@ jobs:
       - name: Run tests
         run: |
           source .venv-pyodide/bin/activate
-          python -m pytest -p no:cacheprovider hypothesis-python/tests/cover
+          # pyodide can't run multiple processes internally, so parallelize explicitly over
+          # discovered test files instead (20 at a time)
+          TEST_FILES=$(python -m pytest -p no:cacheprovider --setup-plan hypothesis-python/tests/cover | grep ^hypothesis-python/ | cut -d " " -f 1)
+          parallel --max-procs 100% --max-args 20 --keep-order --line-buffer \
```

Comment:
With all the other tests parallelized through `xdist`, pyodide was running by itself long after the other tests finish.

This splits the file list to process N=20 files at a time, in parallel, bringing execution time down to the same level as the other tests. The test output is acceptable IMO, although split into 5-6 separate batches.